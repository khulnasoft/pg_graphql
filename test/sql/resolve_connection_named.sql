begin;
    create table account(
        id int primary key
    );


    insert into public.account(id)
    select * from generate_series(1,5);


    select graphql.resolve(
        $$
        query FirstNAccounts($first_: Int!) {
          accountCollection(first: $first_) {
            edges {
              node {
                id
              }
            }
          }
        }
        $$,
        '{"first_": 2}'::jsonb
    );

rollback;
