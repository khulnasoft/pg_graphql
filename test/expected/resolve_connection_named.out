begin;
    create table account(
        id int primary key
    );
    insert into public.account(id)
    select * from generate_series(1,5);
    select graphql.resolve(
        $$
        query FirstNAccounts($first_: Int!) {
          accountCollection(first: $first_) {
            edges {
              node {
                id
              }
            }
          }
        }
        $$,
        '{"first_": 2}'::jsonb
    );
                                        resolve                                         
----------------------------------------------------------------------------------------
 {"data": {"accountCollection": {"edges": [{"node": {"id": 1}}, {"node": {"id": 2}}]}}}
(1 row)

rollback;
